kvbc/include/reconfiguration_kvbc_handler.hpp

```diff
@@ -314,7 +314,8 @@ class InternalPostKvReconfigurationHandler : public concord::reconfiguration::IR
   InternalPostKvReconfigurationHandler(kvbc::IBlockAdder& block_adder, kvbc::IReader& ro_storage)
       : ReconfigurationBlockTools{block_adder, ro_storage} {}
   bool verifySignature(uint32_t sender_id, const std::string& data, const std::string& signature) const override {
-    return true;
+    if (!bftEngine::impl::SigManager::instance()->hasVerifier(sender_id)) return false;
```

Comment:
As an internal handler, this handler should serve only messages which are coming from replicas. Can you here another validation about sender_id is a replica id?